xbmc/interfaces/legacy/RenderCapture.h

```diff
@@ -35,32 +34,36 @@ namespace XBMCAddon
 
     class RenderCapture : public AddonClass
     {
-      CRenderCapture* m_capture;
+      unsigned int m_captureId;
```

Comment:
@FernetMenta who is supposed to set the m_captureId? adapted boblight for the new api but always fail because the capture id is unknown ... and i don't see it getting initialised anywhere too...